src/pattern-library/components/Library.tsx

```diff
@@ -365,12 +369,30 @@ function Usage({ componentName, size = 'md' }: LibraryUsageProps) {
   );
 }
 
+export type LinkProps = {
+  children: ComponentChildren;
+  href: string;
+};
+
+/**
+ * Render an internal link to another pattern-library page.
+ * TODO: Support external links
+ */
+function Link({ children, href }: LinkProps) {
+  return (
+    <RouteLink href={href}>
+      <UILink underline="always">{children}</UILink>
+    </RouteLink>
+  );
+}
```

Comment:
So many things named `Link`, heh.

Reply:
We should call this component `LinkLink`